src/server/graphql/mutations/setOrgUserRole.js

```diff
@@ -81,8 +84,14 @@ export default {
             );
           }),
           updatedAt: now
-        }), {returnChanges: true})('changes')(0)('new_val').default(null)
+        }), {returnChanges: true})('changes')(0)
     });
+    const {old_val: oldOrg, new_val: organization} = organizationChanges;
```

Comment:
Nitpick comment: if we're calling the old organization `oldOrg`, then let's call the new/changed one `newOrg`/`changedOrg`/`updatedOrg` - something like that.

Reply:
i like the parallelism! What if i removed use `oldOrg`? would I then change the name of `newOrg` back to `organization`? Should a variable name be based on the presence of another variable? :thinking:

Reply:
The context here is that there are several organizations, so calling one "organization" is a bit unspecific.